module-4/backend/week_two.md

```diff
@@ -5,18 +5,29 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's one difference between ES5 and ES6?
-2. What's the difference between asynchronous and synchronous JavaScript? 
+Arrow functions are available in ES6, as are classes, both of which are not in ES5
+2. What's the difference between asynchronous and synchronous JavaScript?
+Asynchronous JS allows the use of tools provided in the browser to execute tasks concurrently with the main javascipt thread, that are then rendered once the stack is empty.
 3. What are the four pillars of Object Oriented programming?
+Inheritance, Polymorphism, Abstraction and Encapsulation.
 4. What are some tools available in JavaScript to help you write object oriented code?
+The class tools available in ES6 allow us to create Objects similar to how we would in an Object-Oriented language
```

Comment:
So Javascript is an Object-oriented language it is just not heavily class based. Class tools is good along with `this` and prototype methods